### ISSUE TITLE
After loading the workflow, the main menu disappears in the ComfyUI window

### ISSUE THREAD

**Issue by lednevandrey04** [NimaNzrii/comfyui-photoshop#103]
Hello..
After loading the workflow, the main menu disappears in the ComfyUI window.

![mainmenue](https://github.com/NimaNzrii/comfyui-photoshop/assets/76588218/5d84bafb-3cef-42e4-8383-0027d4f6410b)

Terminal data
--------------------------------------------------------------
Starting server

To see the GUI go to: http://127.0.0.1:8188
FETCH DATA from: D:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\extension-node-map.json [DONE]
_PS_ Node found in workflow: ('127.0.0.1', 60838)
got prompt
Error in is_changed_file for D:\ComfyUI_windows_portable\ComfyUI\input\PS_canvas.png: [Errno 2] No such file or directory: 'D:\\ComfyUI_windows_portable\\ComfyUI\\input\\PS_canvas.png'
Error in is_changed_file for D:\ComfyUI_windows_portable\ComfyUI\input\PS_mask.png: [Errno 2] No such file or directory: 'D:\\ComfyUI_windows_portable\\ComfyUI\\input\\PS_mask.png'
nodepath D:\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-photoshop
model_type EPS
Using pytorch attention in VAE
Using pytorch attention in VAE
Requested to load SD1ClipModel
Loading 1 new model
D:\ComfyUI_windows_portable\ComfyUI\comfy\ldm\modules\attention.py:357: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:455.)
  out = torch.nn.functional.scaled_dot_product_attention(q, k, v, attn_mask=mask, dropout_p=0.0, is_causal=False)
Requested to load AutoencoderKL
Loading 1 new model
Requested to load BaseModel
Loading 1 new model
100%|████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:00<00:00, 13.64it/s]
Error handling connection: sent 1000 (OK); then received 1000 (OK)
Prompt executed in 2.94 seconds

--------------------------------------------------------------
If you load another worker process, the main menu is shown normally.
